tests/unit/StaffNotation.spec.ts

```diff
@@ -272,6 +258,83 @@ describe('StaffNotation', () => {
     wrapper.destroy()
   })
 
+  it('renders drop down menu correctly - historical', async () => {
+    vi.spyOn(utils, 'GetFeatureFlag').mockImplementation(flag => {
+      if (flag === 'supported-restoration-entities') return ''
```

Comment:
If I understand correctly, what this is doing is providing a mock for GetFeatureFlag() but we don't care about the response, because the unit tests below just verify that the menu items are displayed, not whether they are enabled.